includes/blocks/safe-svg/frontend.scss

```diff
@@ -1,15 +1,33 @@
-.safe-svg-cover {
-    text-align: center;
-
-    .safe-svg-inside {
-        display: inline-block;
-        max-width: 100%;
-    }
-
-    svg {
-        height: 100%;
-        max-height: 100%;
-        max-width: 100%;
-        width: 100%;
-    }
+.wp-block-safe-svg-svg-icon	{
+	svg {
+		display: block;
+		height: auto;
+		max-block-size: 100%;
+		max-inline-size: 100%;
+	}
+
+	&.alignfull,
+	&.alignwide {
+		svg {
+			width: 100%;
+		}
+	}
+
+	&.aligncenter,
+	&.alignleft,
+	&.alignright {
+		display: table;
+	}
+
+	&.aligncenter {
```

Comment:
If we're trying to sync the SVG block up with the image block, the image block use float and margin rather than text-align for `align*` styles.

https://github.com/WordPress/WordPress/blob/master/wp-includes/css/dist/block-library/style.css#L1129C1-L1139C2

Reply:
Good point, I think I saw this and just assumed it was `text-align` for everything
https://github.com/WordPress/WordPress/blob/master/wp-includes/css/dist/block-library/style.css#L1114

I noticed that the global inline styles included floats & margin for `align*` styles, which is what I see being used for the image block and the SVG block in the inspector. The `align*` class gets applied to the image/SVG block itself, and the rulesets in the image block styles look like they're for child elements with `align*` classes.
<img width="350" alt="Screenshot 2024-07-26 at 10 47 04 AM" src="https://github.com/user-attachments/assets/24143f4d-b76d-45cf-acf2-f9f925e05277">

Reply:
Ahh ok, that makes sense.